### ISSUE TITLE
KISS Newline 192 c0 Bug

### ISSUE THREAD

**Issue by na7q** [ge0rg/aprsdroid#360]
So after some debugging with VARA and the developer. Jose has discovered that APRSDroid creates a new line after the first byte of c0.

Instead of the KISS data being one single line, it's c0, newline, then the rest of the data.

**Comment by na7q** [comment]
I have a script that shows the raw data from APRSDroid. As you can see, two lines, followed by the actual packet itself. c0 should be with data seen on line 2. Creating 1 line. 

13:39:11: c0
13:39:11: 00 82 a0 88 a4 62 6a e0 9c 82 6e a2 40 40 66 40 40 40 40 40 40 61 03 f0 3d 34 36 31 30 2e 32 34 4e 2f 31 32 33 33 34 2e 32 35 57 52 20 c0
13:39:11: NA7Q-3>APDR15:=4610.24N/12334.25WR

**Comment by pepefrog1234** [comment]
I have found this problem before, the app will crash VARA after sending kiss packs again.

**Comment by na7q** [comment]
> I have found this problem before, the app will crash VARA after sending kiss packs again.

Indeed. Jose (VARA dev) is aware, I have already tested the unreleased version of VARA that fixes this. It should be released within the next day or two. We are working on a couple other minor fixes for the HF modem. I believe the FM version is fixed and released already.

**Comment by pepefrog1234** [comment]
VARA authors have released a new version.

**Comment by na7q** [comment]
def writePacket(p : APRSPacket) {
		Log.d(TAG, "writePacket: " + p)
		val combinedData = Array[Byte](Kiss.FEND.toByte, Kiss.CMD_DATA.toByte) ++ p.toAX25Frame() ++ Array[Byte](Kiss.FEND.toByte)
		os.write(combinedData)
		os.flush()
	}
